lib/chef/provisioning/aws_driver/driver.rb

```diff
@@ -498,6 +535,53 @@ def default_ami_for_region(region)
       end
     end
 
+    def create_winrm_transport(machine_spec, machine_options, instance)
+      remote_host = determine_remote_host(machine_spec, instance)
+      puts "remote host: #{remote_host}"
```

Comment:
Yeah, oops, will go through it again. I'm fresh into the ruby world, is there an easier way to debug this other than old-school print-to-console logging?